pycoin/Blockchain.py

```diff
@@ -265,7 +274,7 @@ def merkle_path(self, transaction: Transaction):
 
     def search_block_with_transaction(self, transactionHash):
         '''return block that matches given transaction hash'''
-        fullchain = [json.loads(block) for block in self.chain]
+        fullchain: List[Any] = [json.loads(block) for block in self.chain]
```

Comment:
actually this should be `List[dict]` rather than List[Any]. But I recommend you to just leave it without type annotation.